Makefile

```diff
@@ -400,6 +427,8 @@ else ifeq ($(TARGET), pocketbook)
 	make pbupdate
 else ifeq ($(TARGET), sony-prstux)
 	make sony-prstuxupdate
+else ifeq ($(TARGET), cervantes)
+        make cervantesupdate
```

Comment:
This actually works with this indentation?

I mean, change it to tab regardless, but I'm slightly surprised that it does.

Reply:
@Frenzie: not sure, I just pulled from master and had a conflict in the Makefile, so I just ditch changes and copy-paste from old copy :p